yml/apdb.yaml

```
@@ -1570,6 +1561,16 @@ tables:
     description: Estimated uncertainty of apFlux.
     fits:tunit: nJy
     ivoa:ucd: stat.error;phot.count
+  - name: apFlux_flag
+    "@id": "#DiaSource.apFlux_flag"
+    datatype: boolean
+    description: General Failure Flag
```

Similar to the centroid one, lets make this more useful,.

```suggestion
    description: General aperture flux algorithm failure flag; set if anything went wrong when measuring aperture fluxes. Another apFlux flag field should also be set to provide more information.
```